.nvmrc

```diff
@@ -0,0 +1 @@
+8
```

Comment:
The + is from the diff. The file only contains `8`, which is the correct syntax (It just pulls the latest stable 8 that nvm can grab)